app/views/layouts/admin.html.erb

```diff
@@ -44,6 +44,7 @@
                 <li><%= link_to 'Actions', main_app.routes.url_helpers.actions_admin_users_path %></li>
                 <li><%= link_to 'Pre-Auth States', main_app.routes.url_helpers.admin_pre_auth_states_path %></li>
                 <li><%= link_to 'Reports', main_app.routes.url_helpers.admin_reports_path %></li>
+                <li><%= link_to 'Student Accounts Report', main_app.routes.url_helpers.actions_admin_student_accounts_path %></li>
```

Comment:
I think that creating a new page and everything for just this one report isn't necessary. To me it seems like the `'Reports'` page was perfect for that. Here's what it currently looks like: 

![Screen Shot 2020-01-28 at 10 41 58 AM](https://user-images.githubusercontent.com/6620941/73284907-da468a80-41ba-11ea-96f4-68d7910f0559.png)

This report, then, would be just one extra line :).

Reply:
@BryanEliDimas I did not add this to the existing user report because the use case is different. The existing report is about user totals. I added a report for student accounts created since July 1. I thought mixing the two could be confusing.

Reply:
I think it is probably ok to add it to the user reports page, because it is reporting on "student users", just as how the other sections here are about "general users" and "faculty users".  

So a heading of "Students" with a line item of "New since July 1" seems reasonable.

Reply:
Report moved to User report